presto/dataset.py

```diff
@@ -32,7 +34,7 @@
 
 
 class WorldCerealBase(Dataset):
-    _NODATAVALUE = 65535
+    # _NODATAVALUE = 65535
```

Comment:
@cbutsko why is this commented? And are changes below like the addition of `get_timestep_positions` not part of the other branch?